package.json

```diff
@@ -69,7 +69,8 @@
   },
   "devDependencies": {
     "@seamapi/types": "1.184.0",
-    "@types/node": "^20.8.10",
+    "@types/eslint": "^8.56.10",
```

Comment:
Why was this added? And why did the other dependencies need explicit updates, was something causing an issue?

Reply:
Didn't notice these were already present.